ios/RNFirebase/notifications/RNFirebaseNotifications.m

```diff
@@ -105,12 +105,22 @@ - (void)didReceiveLocalNotification:(nonnull UILocalNotification *)localNotifica
         void (^fetchCompletionHandler)(UIBackgroundFetchResult) = fetchCompletionHandlers[handlerKey];
         if (fetchCompletionHandler != nil) {
             fetchCompletionHandlers[handlerKey] = nil;
-            fetchCompletionHandler(fetchResult);
+            @try {
+                fetchCompletionHandler(fetchResult);
+            }
+            @catch (NSException * e) {
+                NSLog(@"Exception fetchCompletionHandler: %@", e);
+            };
         } else {
             void(^completionHandler)(void) = completionHandlers[handlerKey];
             if (completionHandler != nil) {
                 completionHandlers[handlerKey] = nil;
-                completionHandler();
```

Comment:
I think this line should not be deleted, because it would cause the app to crash